### ISSUE TITLE
Design of predict sub-dialog

### ISSUE THREAD

**Issue by stevekogo** [africanmathsinitiative/Instat#2246]
This sub-dalog is needed by the use model dialogues. Could we have a design of how it should look like?

**Comment by rdstern** [comment]
You are right that we now need to "attack" the use model dialogues.  I still don't have a very clear idea of how it will look.  I think I know the sort of things I want.  The fit model and choose model dialogue (2 variable up to general) will produce regression objects.  The use model selector will use those objects.  

Part of my confusion is because we are trying to cope with such a wide range of types of data for the y-variable in the single dialogue.  I think that is good in a way.  But we must be careful we don't pay too high a price, either in the dialogue becoming too complicated for our users, or in the results not being displayed in ways that would only be appropriate for one type of data.

It seems obvious that we will still allow a look at the residuals, but the fitted value graphs become more important.   And we will want predict to also give fitted value tables.  These can be at the different factor levels for the factors, and at specified example points for variates.

I hope this isn't a diversion, but we also want to display the results from fitted models clearly.  I think (at this stage) this probably comes best in our choose model dialogues, but we may want to asses this as we go along.  A related point on all these dialogues Fit, Choose, Use (I think) is to display the results in better ways than the default output from lm or glm etc.  

I wrote a long time ago - March - about the possibility of using (for example the stargazer package and I would still like to consider this.  

I think what is needed is for us to spend some time in RStudio with all these, before trying to get to specific menus.  And we need to document carefully, because I am not sure we can "finish" before turning attention (at least briefly) to the climatic dialogues.

**Comment by dannyparsons** [comment]
@rdstern this still looks useful, tagging as 0.9.x (post July)

**Comment by lilyclements** [comment]
Closing for now. Mainly because we will be discussing this in more detail elsewhere when we get to our modelling dialogs. Adding Roger's "Return Later" tag
